src/main/java/io/jenkins/plugins/audit/RequestContext.java

```diff
@@ -80,7 +79,7 @@ public static String getRequestId() {
             uuid = UuidUtil.getTimeBasedUuid();
             ThreadContext.put(REQUEST_ID, uuid.toString());
         }
-        return uuidStr;
+        return ThreadContext.get(REQUEST_ID);
```

Comment:
This change is unnecessary. `uuidStr` already contains the updated value.

Reply:
but if `uuidStr` is null then null will be returned

Reply:
Oh, good point. Then maybe you could reassign uuidStr inside the if block. You don't even need the UUID variable; you can convert it straight to a string right away.